packages/ui/src/Layout/AdminBar.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import MobileHamburgerMenu from "./MobileHamburgerMenu";
+
+interface Item {
+  Icon: string;
+  label: string;
+  slug: string;
+}
+interface AdminBarProps {
+  isAdmin?: boolean;
+  adminLinks: Item[];
+}
+
+const AdminBar: React.FC<AdminBarProps> = ({ isAdmin, adminLinks }) => {
+  return (
+    <div>
+      <MobileHamburgerMenu adminLinks={adminLinks} />
+      <div className={getHeaderRowClasses()}>
+        {isAdmin &&
+          adminLinks.map(({ Icon, label, slug }, i) => {
+            return (
+              <div
```

Comment:
Using a ‘Button’ component here would solve a couple of problems (hover:cursor-pointer, and avoid the div soup, plus it has a bunch of predetermined styles). Icon can be passed as ‘inputAdornment’.

Reply:
Additionally Link should encapsulate the button (not just the label)